### PR TITLE
refactor: remove fee struct

### PR DESCRIPTION
**Pull request by Roznovjak**

### CODE REVIEW

**Review by enthusiastmartin — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -6,7 +6,7 @@ license = 'Apache-2.0'
 name = "hydra-dx-math"
 description = "A collection of utilities to make performing liquidity pool calculations more convenient."
 repository = 'https://github.com/galacticcouncil/hydradx-math'
-version = "4.2.0"
+version = "4.2.1"
```
Comment (jak-pan) [inline Cargo.toml:9]: This is a breaking change

Comment (Roznovjak) [reply Cargo.toml:9]: Do we want to create a new PR for this?

Comment (enthusiastmartin) [reply Cargo.toml:9]: it is not breaking change. the fee struct was only used internally in the math crate.

Comment (enthusiastmartin) [reply Cargo.toml:9]: and it wasn't  doing anything useful anymore.

Comment (jak-pan) [reply Cargo.toml:9]: > it is not breaking change. the fee struct was only used internally in the math crate.

Oh. Then sorry I was sure I saw this on the frontend somewhere but I might have been wrong.

**Pull request merged**